cl_dll/hud_redraw.cpp

```diff
@@ -127,6 +128,9 @@ int CHud :: Redraw( float flTime, int intermission )
 			// Take a screenshot if the client's got the cvar set
 			if ( CVAR_GET_FLOAT( "hud_takesshots" ) != 0 )
 				m_flShotTime = flTime + 1.0;	// Take a screenshot in a second
+
+			if ( CVAR_GET_FLOAT( "cl_autorecord_stop") != 0 && CVAR_GET_FLOAT( "cl_autorecord") != 0)
```

Comment:
Not heavily related to `cl_autorecord`, no need to check if it's enabled.
```suggestion
			if ( CVAR_GET_FLOAT( "cl_autorecord_stop") != 0)
```